### DISCUSSION TITLE
Plans to address recent deprecation warnings?

### DISCUSSION THREAD

**Discussion by aliakb** [aws-actions/configure-aws-credentials#539] [General]
Hello,

I started getting two deprecation warnings from Github actions about the `aws-actions/configure-aws-credentials@v1` action. The warnings are: 

1. Node.js 12 actions are deprecated.
2. The 'set-output' command is deprecated and will be disabled soon.

Could you share what your plans to address these are? Thanks!

**Comment by peterwoodworth** [comment]
Hey @aliakb, thanks for bringing this up. 

You will want to use `@v1-node16` if you want to get rid of the deprecation warning about node 12. The `set-output` warning should be gone now on both branches, let me know if it isn't. See [this comment](https://github.com/aws-actions/configure-aws-credentials/issues/489#issuecomment-1278145876) (and thread) for why we moved to a new branch to use node 16, and for more information on our future plans 🙂

**Comment by peterwoodworth** [comment]
Update: use `@v2`. Thanks!
